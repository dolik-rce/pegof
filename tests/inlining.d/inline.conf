input inlining.d/inline.peg
optimize
