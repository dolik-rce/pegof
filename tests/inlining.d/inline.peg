X <- A+ B* C? D
A <- "A"* "a"
B <- "B"+
C <- [Cc]*
D <- !.
