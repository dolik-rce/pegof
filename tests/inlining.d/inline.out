X <- ("A"* "a")+ ("B"+)* ([Cc]*)? !.
